From 6113cf7afa0e80d8fbf51a9700f9a1fd481c2600 Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:54:21 -0400
Subject: [fs] ext4: Add __init marking to init_inodecache

Message-id: <1414058125-4183-108-git-send-email-lczerner@redhat.com>
Patchwork-id: 98208
O-Subject: [RHEL 7.1 PATCH v2 107/171] ext4: Add __init marking to init_inodecache
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Fabian Frederick <fabf@skynet.be>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit e67bc2b35905cb82e9ee1f485095d8c0739b68c8

 init_inodecache is only called by __init init_ext4_fs.

 Signed-off-by: Fabian Frederick <fabf@skynet.be>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/super.c b/fs/ext4/super.c
index 55b733f..ea3adc2 100644
--- a/fs/ext4/super.c
+++ b/fs/ext4/super.c
@@ -944,7 +944,7 @@ static void init_once(void *foo)
  inode_init_once(&ei->vfs_inode);
 }
 
-static int init_inodecache(void)
+static int __init init_inodecache(void)
 {
  ext4_inode_cachep = kmem_cache_create("ext4_inode_cache",
           sizeof(struct ext4_inode_info),
-- 
1.7.1